modules/overrides/runtime_config_overrides.go

```diff
@@ -109,7 +109,7 @@ func newRuntimeConfigOverrides(cfg Config) (Service, error) {
 			ReloadPeriod: time.Duration(cfg.PerTenantOverridePeriod),
 			Loader:       loadPerTenantOverrides(cfg.ConfigType),
 		}
-		runtimeCfgMgr, err := runtimeconfig.New(runtimeCfg, prometheus.WrapRegistererWithPrefix("tempo_", prometheus.DefaultRegisterer), log.Logger)
+		runtimeCfgMgr, err := runtimeconfig.New(runtimeCfg, "overrides", prometheus.WrapRegistererWithPrefix("tempo_", prometheus.DefaultRegisterer), log.Logger)
```

Comment:
what's the new string do?

Reply:
I think it makes it so there can be more than one runtime manager.  I'm still checking on the tests to confirm we don't have a change in behavior.  I think if we just name it anything that matches the intent, then everything will be internally consistent.